Revert "request delivery reports by default"

pR0Ps opened the pull request:
This reverts commit b0e1020fd2da32352edbb22709fe0edaa0f2eee8.

Some mobile networks charge for delivery reports so this option
shouldn't be on by default.

BastienLQ commented:
I rebased on `master` and added a commit with a reminder for delivery reports. Thoughts?

pR0Ps commented:
Looks good to me!

On 12 November 2016 at 19:20, Bastien Le Querrec <notifications@github.com>
wrote:

> I rebased on master and added a commit with a reminder for delivery
> reports. Thoughts?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SilenceIM/Silence/pull/440#issuecomment-260157676>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAcf_QPltcOxE5lubXQ_mmnAEBojrv9Tks5q9leygaJpZM4Ju7Na>
> .
>
